great_expectations/rule_based_profiler/data_assistant_result/data_assistant_result.py

```diff
@@ -60,6 +71,11 @@
 ColumnDataFrame = namedtuple("ColumnDataFrame", ["column", "df"])
 
 
+def default_field(obj: Dict[Union[str, Tuple[str]], str]) -> field:
```

Comment:
It seems like this function is sprinkled across a few files. I think it would be better in an util file that we import.

Reply:
✅